### PR TITLE
cool article, some proposed changes

### PR DESCRIPTION
**Pull request by lionelmoi**

### CODE REVIEW

File: Custom Url Schemes/README.md [hunk 1]
```diff
@@ -103,9 +103,9 @@ At the end of the file, before the closing tags `</dict></plist>`, insert the fo
 </array>
 ```
 
-**Notice** the string `lumapps-vscode`, it's your custom scheme.
+**Notice** the string `lumapps-vscode`, this is your custom scheme.
```
Comment (maximevast) [inline Custom Url Schemes/README.md:106]: Can you please apply this change to the two other Operating Systems?

File: Custom Url Schemes/README.md [hunk 2]
```diff
@@ -9,19 +9,19 @@ A Uniform Resource Identifier ([URI](https://en.wikipedia.org/wiki/Uniform_Resou
...
 
-Most modern softwares comes with pre-defined custom schemes; that allow you to interact with the software through URL.
+Most modern softwares comes with pre-defined custom schemes, that allow you to interact with the software through URL.
 
 [Microsoft Office](https://docs.microsoft.com/en-us/office/client-developer/office-uri-schemes)™ productivity applications for instance, allows you to open a document in *edit|view|from_template* modes using *ms-word, ms-powerpoint, ms-excel, ms-visio, ms-access, ms-project, ms-publisher, ms-spd or ms-infopath*.
 
 ## Defining custom schemes
 
-If your favorite software does not provide a custom url scheme, you can define it yourself. In the following examples, we will be using Microsoft Visual Studio code, but the explanations should work for *any* software.
+If your favorite software does not provide a custom URL scheme, you can define it yourself. In the following examples, we will use Microsoft Visual Studio code, but this explanations can work for *any* software.
```
Comment (maximevast) [inline Custom Url Schemes/README.md]: this explanation or these explanations

**Review by maximevast — CHANGES_REQUESTED** [review]

File: Custom Url Schemes/README.md (hunk 2 above)
Comment (lionelmoi) [reply Custom Url Schemes/README.md]: these explanations

File: Custom Url Schemes/README.md (hunk 1 above)
Comment (lionelmoi) [reply Custom Url Schemes/README.md:106]: done

File: Custom Url Schemes/README.md (hunk 2 above)
Comment (lionelmoi) [reply Custom Url Schemes/README.md]:
```suggestion
If your favorite software does not provide a custom URL scheme, you can define it yourself. In the following examples, we will use Microsoft Visual Studio code, but these explanations can work for *any* software.
```

**Pull request merged**